### PR TITLE
Add more launcher config options such as title and button colors

### PR DESCRIPTION
**Pull request by Medik-GH**

### CODE REVIEW

File: M2TWEOP Code/M2TWEOP GUI/dataG.h [hunk 1]
```diff
@@ -26,6 +26,11 @@ class dataG
 			string exeName;
 			string gameArgs;
 
+			// Customization Options
```
Comment (yaneay) [inline M2TWEOP Code/M2TWEOP GUI/dataG.h:29]: set default values here(0 for ints ) and empty string for modTitle

File: M2TWEOP Code/M2TWEOP GUI/managerG.cpp [hunk 2]
```diff
@@ -66,104 +77,121 @@ namespace managerG
...
+			if (json.contains("modTitle"))
 			{
-				getline(f1, s);
-				dataG::data.audio.bkgMusic.musicVolume = stoi(s);
+				getJson(dataG::data.gameData.modTitle, "modTitle");
 			}
+			//if (json.contains("runButtonColor"))
+			//{
+				// getJson(jsonStringValue, "buttonColor");
+				jsonStringValue = "f024b6";
```
Comment (yaneay) [inline M2TWEOP Code/M2TWEOP GUI/managerG.cpp]: add FF to end. You set rgb here only, not rgba

File: M2TWEOP Code/M2TWEOP GUI/managerG.cpp [hunk 3]
```diff
@@ -66,104 +77,121 @@ namespace managerG
...
+				dataG::data.gameData.buttonColor = stoi(jsonStringValue, NULL, 16); // Convert to hex
+		//	}
+			//if (json.contains("runButtonHoverColor"))
+			//{
+				// getJson(jsonStringValue, "runButtonHoverColor");
+				jsonStringValue = "cfbfe5";
```
Comment (yaneay) [inline M2TWEOP Code/M2TWEOP GUI/managerG.cpp]: add FF to end. You set rgb here only, not rgba

File: .gitignore [hunk 4]
```diff
@@ -345,10 +345,3 @@ ASALocalRun/
 # BeatPulse healthcheck temp database
 healthchecksdb
 
-
```
Comment (yaneay) [inline .gitignore:348]: why remove it?

Comment (Medik-GH) [reply .gitignore:348]: Mistake

**Pull request merged**